staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -762,8 +763,10 @@ func DefaultBuildHandlerChain(apiHandler http.Handler, c *Config) http.Handler {
 	handler = genericapifilters.WithWarningRecorder(handler)
 	handler = genericapifilters.WithCacheControl(handler)
 	handler = genericfilters.WithHSTS(handler, c.HSTSDirectives)
+	handler = httplog.WithLogging(handler, httplog.DefaultStacktracePred)
 	handler = genericapifilters.WithRequestReceivedTimestamp(handler)
 	handler = genericfilters.WithPanicRecovery(handler, c.RequestInfoResolver)
+	handler = genericapifilters.WithAuditID(handler)
```

Comment:
It's before `WithPanicRecovery` - this is so we can access the audit ID from the context when we log error message inside the `WithPanicRecovery`  handler. 
If `WithAuditID` panics, `net/http2` will catch it and then log the panic with stack trace.

Reply:
sounds fine